fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaStatisticProvider.java

```diff
@@ -119,7 +119,12 @@ public Statistics getTableStatistics(DeltaLakeTable deltaLakeTable,
     }
 
     public Map<ColumnRefOperator, ColumnStatistic> buildUnknownColumnStatistics(Set<ColumnRefOperator> columns) {
-        return columns.stream().collect(Collectors.toMap(column -> column, column -> ColumnStatistic.unknown()));
+        return columns.stream().collect(Collectors.toMap(column -> column, column -> ColumnStatistic.builder()
```

Comment:
Is same?
```suggestion
        return columns.stream().collect(Collectors.toMap(column -> column, column -> ColumnStatistic.unknown())
```

Reply:
private static final ColumnStatistic UNKNOWN =

is one static final instance.